frontend/test/metabase/scenarios/filters/helpers/e2e-sql-fitler-helpers.js

```diff
@@ -0,0 +1,103 @@
+import { filterWidget, popover } from "__support__/e2e/cypress";
+
+/****************************************************************
+ *                        FILTER TYPES                          *
+ ****************************************************************/
+
```

Comment:
I am not crazy about banner comments.

This may be merely a personal opinion, but I observe that, when reading code, they call attention more to themselves than to the code.

In js, I don't remember seeing them used in any codebase I touched, so this may compound the effect of strangeness.

So if we want to have markers by "subject", merely using uppercase comments may do the trick.

As in 

~~~js
// FILTER TYPES
~~~

Reply:
Thanks for the comment and the suggestion, Gus.
I was worried that distinction between sections would get lost because of all the JSDoc annotations. Changed it to `// UPPERCASED COMMENT`.
Please check how it looks now.

FWIW, I can remove banners completely.